runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -807,54 +989,85 @@ TR_VectorAPIExpansion::validateSymRef(int32_t id, int32_t i, vec_sz_t &classLeng
    if (_aliasTable[i].*classField == -1)
       {
       if (_trace)
-         traceMsg(comp(), "%s invalidating1 class #%d due to symref #%d\n", OPT_DETAILS_VECTOR, id, i);
+         traceMsg(comp(), "%s invalidating12 class #%d due to symref #%d\n", OPT_DETAILS_VECTOR, id, i);
       return false;
       }
    else if (symRef->getSymbol()->isShadow() ||
-             symRef->getSymbol()->isStatic() ||
+            symRef->getSymbol()->isStatic() ||
              symRef->getSymbol()->isParm())
       {
+      if (boxingAllowed())
+         {
+         _aliasTable[i]._vecLen = vec_len_boxed_unknown;
+         _aliasTable[id]._vecLen = vec_len_boxed_unknown;
+         return true;
+         }
+
       if (_trace)
-         traceMsg(comp(), "%s invalidating2 class #%d due to symref #%d\n", OPT_DETAILS_VECTOR, id, i);
-      return false;
+         traceMsg(comp(), "%s invalidating13 class #%d due to symref #%d\n", OPT_DETAILS_VECTOR, id, i);
+         return false;
```

Comment:
Please revert the indentation of this `return` line.

Reply:
done